packages/off-chain/outputs/merkleTree.json

```diff
@@ -0,0 +1,782 @@
+{
+    "tree": {
```

Comment:
Where exactly does the output from `tree.dum()` start?

Reply:
everything from `tree.dump` is stored in this property.